SCons/Util.py

```diff
@@ -2138,6 +2138,33 @@ def format(self, record):
         formatter = self._formatters.get(record.name, self._default_formatter)
         return formatter.format(record)
 
+
+def sanitize_shell_env(env):
+    """
+    Sanitize all values from env['ENV'] which will be propagated to the shell
```

Comment:
I can supply an updated docstring if interested (haven't stopped to write one up yet). Among other things it's a little confusing that the param is `env`, then we say "all values from `env['ENV']`, then we proceed not to look at that. Yes I know they're different env's - the one in the first docstring line is the abstract "current Environment", not the parameter name.

Reply:
that would work